src/breeze/components/breeze/pagination.cr

```diff
@@ -1,4 +1,4 @@
-class Breeze::PageNav < BreezeComponent
+class Breeze::Pagination < BreezeComponent
```

Comment:
I guess this could have stayed `PageNav`, but `Pagination` just seemed more common to me. I'm cool with either way if anyone has a preference.